lib/Optimizer/GraphOptimizer/ConstantFolding.cpp

```diff
@@ -302,6 +302,12 @@ bool glow::ConstantFold::run(Function *F, const CompilationContext &cctx) {
     if (!hasNonConstantOperationUser(N, *backend)) {
       continue;
     }
+    // Don't try to constant fold Nodes that are not supported by the
+    // Interpreter. These are usually backend-specific.
+    if (!backend->isOpSupported(*N)) {
```

Comment:
nit: Could probably move this up in the loop, as the above two calls are a bit more complex.